.github/workflows/_build_base.yaml

```diff
@@ -77,8 +87,12 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
-            BASE_IMAGE=${{ inputs.BASE_IMAGE }}
             BUILD_DATE=${{ inputs.BUILD_DATE }}
+            ${{
```

Comment:
Give that this is an reusable workflow, i.e. an internal function called by other 'frontend' workflows, it is better to just pass in `inputs.BASE_IMAGE` and let the job error out if the value is invalid